Devices/Lumia/950s/Talkman/Main.txt

```diff
@@ -0,0 +1,185 @@
+}
+
+PrepareDisk
+{
+	'Preparing partition layout'
+	
+	RemovePartitionSafe("Disk={Disk}, Name='MSR'");
+	RemovePartitionSafe("Disk={Disk}, Name='Windows'");
+	RemovePartitionSafe("Disk={Disk}, Name='SYSTEM'");
+	
+	if (GetDiskSize(Disk) < SizeFromString("{DeploymentSize}GB"))
+	{
+		'Trying to allocate required space from the Data partition'
+
+		data = "Disk={Disk}, Name='Data'";
+		if (PartitionExists(data)) 
+		{
+			'Allocating space'
+			AllocateSpace(data, "{DeploymentSize}GB");	
+		}	
+		else 
+		{
+			Fail("Could not allocate the required space in the phone. We need at least {DeploymentSize}GB of free space to deploy Windows");
+		}
+	} 
+		
+	systemPart = CreateGptPartition(Disk, "Esp", "SYSTEM", "100MB");
+	CreateGptPartition(Disk, "Reserved", "MSR", "16MB");
+	windowsPart = CreateGptPartition(Disk, "Basic", "Windows", "*");
+	
+	systemRoot = GetPartitionRoot(systemPart);
+	windowsRoot = GetPartitionRoot(windowsPart);
+	
+	Format(systemPart, "Fat32", "SYSTEM");
+	Format(windowsPart, "Ntfs", "Windows");
+}
+
+DeployWindows
+{
+	'Deploying Windows'
+
+	ApplyWindowsImage(WimFilePath, WimFileIndex, windowsRoot);
+	MakeWindowsBootable("{systemRoot}", "{windowsRoot}\Windows");
+	SetGptType("{systemPart}", "Esp");
+}
+
+InstallBootShim
+{
+	'Downloading UEFI Loader (BootShim)'
+	Unzip(AzureDevOpsArtifact("LumiaWOA", "Boot Shim", 3, "UEFI Loader for 8992 and 8994"), "{packages}\UEFI Loader");
+
+	'Installing UEFI Loader'
+	Copy("{packages}\UEFI Loader\artifacts\UEFI Loader for 8992 and 8994\ARM\Debug\BootShim.efi", "{efiEspRoot}\EFI\Boot\BootShim.efi");
+}
+
+ConfigureBcd
+{
+	'Configuring Boot'
+	
+	bcdWoa = "7619dcca-fafe-11d9-b411-000476eba25f";
+	bcdWinMobile = "7619dcc9-fafe-11d9-b411-000476eba25f";
+	efiBcdStore = "{efiEspRoot}\EFI\Microsoft\Boot\BCD";
+	
+	BcdEditCreate(efiBcdStore, bcdWoa, "/d ""Windows 10"" /application BOOTAPP");
+	BcdEdit(efiBcdStore, "/set {{{bcdWoa}}} path \EFI\boot\BootShim.efi");
+	BcdEdit(efiBcdStore, "/set {{{bcdWoa}}} device partition={efiEspRoot}");
+	BcdEdit(efiBcdStore, "/set {{{bcdWoa}}} nointegritychecks on");
+	
+	BcdEdit(efiBcdStore, "/set {{{bcdWinMobile}}} path dummy");
+	BcdEdit(efiBcdStore, "/set {{{bcdWinMobile}}} description ""Dummy, please ignore""");
+	
+	BcdEdit(efiBcdStore, "/set {{bootmgr}} displaybootmenu on");
+	BcdEdit(efiBcdStore, "/deletevalue {{bootmgr}} customactions");
+	BcdEdit(efiBcdStore, "/deletevalue {{bootmgr}} processcustomactionsfirst");
+	
+	BcdEdit(efiBcdStore, "/displayorder {{{bcdWoa}}}");
+	BcdEdit(efiBcdStore, "/displayorder {{{bcdWinMobile}}} /addlast");
+	BcdEdit(efiBcdStore, "/default {{{bcdWoa}}}");
+	BcdEdit(efiBcdStore, "/timeout 30");
+}
+
+AddDrivers
+{
+	'Downloading drivers'
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "HARDWARE.INPUT.SYNAPTICS_RMI4_F12_WIN10.zip"), packages);
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "DEVICE.SOC_QC899X.CITYMAN.zip"), packages);
```

Comment:
Should be DEVICE.SOC_QC899X.TALKMAN.zip instead, you probably missed that one :)